Solutions/HolmSecurity/Data Connectors/HolmSecurityAssets_REST_API_Connector.json

```diff
@@ -0,0 +1,149 @@
+        },
+        {
+            "title": "",
+            "description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+        },
+        {
+            "title": "",
+            "description": "**STEP 1 - Configuration steps for the HOLM Security API**\n\n [Follow these instructions](https://support.holmsecurity.com/hc/en-us/articles/360027651591-How-do-I-set-up-an-API-token-) to create an API authentication token."
+        },
+        {
+            "title": "",
+            "description": "**STEP 2 - Use the below deployment option to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Holm Security connector, have the Workspace ID  and Workspace Primary Key (can be copied from the following), as well as the Holm Security API authorization Token, readily available.",
+            "instructions": [
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                },
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "PrimaryKey"
+                        ],
+                        "label": "Primary Key"
+                    },
+                    "type": "CopyableLabel"
+                }
+            ]
+        },
+        {
+            "title": "Azure Resource Manager (ARM) Template Deplyment",
+            "description": "**Option 1 - Azure Resource Manager (ARM) Template**\n\nUse this method for automated deployment of the HOLM Security connector.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinel-holmsecurityassets-azuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter the **Workspace ID**, **Workspace Key**, **API Username**, **API Password**, 'and/or Other required fields'. \n>Note: If using Azure Key Vault secrets for any of the values above, use the`@Microsoft.KeyVault(SecretUri={Security Identifier})`schema in place of the string values. Refer to [Key Vault references documentation](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) for further details. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
@NikitaGrunskyHolm , After deployment of this azure function, getting following error.
'Microsoft.Azure.WebJobs.Script: Did not find functions with language [python].'

Can you please check and fix? Thanks!

![image](https://user-images.githubusercontent.com/75478517/134349765-16f8dcff-b414-46bd-b834-153e915a479b.png)

Reply:
I think it's because I've forgot to add send_net_assets and send_web_assets functions into main function.

Reply:
@NikitaGrunskyHolm , We are still getting this error after deployment. Please let us know if any other instrucion we need follow. Thanks!
Please refer existing[ Cisco Umbrella python azure function](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/CiscoUmbrella) for refernce.

Similar issue is there in azure function github repo.
https://github.com/Azure/Azure-Functions/issues/1511

Reply:
@v-maudan , Are the tests working fine? I rollback changes to this https://github.com/Azure/Azure-Sentinel/pull/2939/commits/a3b4678dd6360f1ee27b71ed29321bb7263a48bd commit, but tests fail. You suggested me refer Cisco Umbrella for reference, which storage files in DataConnectors folder, but previously @NikTripathi and @v-rucdu asked me to move all files into Solutions folder. So which folder is correct? I used this guide https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/README.md

Reply:
@NikitaGrunskyHolm , It should be part of solution folder, please revert this changes.